packages/fx-core/resource/package.nls.json

```diff
@@ -827,27 +825,20 @@
   "depChecker.finishInstallFunctionCoreTool": "Successfully installed @NameVersion.",
   "depChecker.needReplaceWithFuncCoreToolV3": "You must update to @NameVersion to debug your local functions.",
   "depChecker.needInstallFuncCoreTool": "You must have @NameVersion installed to debug your local functions.",
-  "depChecker.failToInstallFuncCoreTool": "Unable to install @NameVersion. Install @NameVersion manually.",
   "depChecker.failToValidateFuncCoreTool": "Unable to validate @NameVersion after installation.",
   "depChecker.globalFuncNodeNotMatched": "Your Node.js (@NodeVersion) is incompatible with your Azure Functions Core Tools (@FuncVersion). Please make sure your Azure Functions Core Tools works with your Node.js version. See (@link) for Azure Functions supported Node.js versions.",
   "depChecker.portableFuncNodeNotMatched": "Your Node.js (@NodeVersion) is incompatible with Teams Toolkit Azure Functions Core Tools (@FuncVersion). Go to @Link to install Node.js (@RecommendedVersion is recommended).",
   "depChecker.startInstallNgrok": "Downloading and installing @NameVersion.",
   "depChecker.finishInstallNgrok": "Successfully installed @NameVersion.",
   "depChecker.finishInstallBicep": "Successfully installed @NameVersion.",
   "depChecker.needInstallNgrok": "You must have @NameVersion installed to debug your local bot.",
-  "depChecker.failToInstallNgrok": "Unable to install @NameVersion. Install @NameVersion manually.",
   "depChecker.failToValidateNgrok": "Unable to validate @NameVersion after installation.",
   "depChecker.downloadDotnet": "Downloading and installing the portable version of @NameVersion, which will be installed to @InstallDir and will not affect your environment.",
   "depChecker.downloadBicep": "Downloading and installing the portable version of @NameVersion, which will be installed to @InstallDir and will not affect your environment.",
   "depChecker.finishInstallDotnet": "Successfully installed @NameVersion.",
   "depChecker.useGlobalDotnet": "Using dotnet from PATH:",
   "depChecker.dotnetInstallStderr": "dotnet-install command failed without error exit code but with non-empty standard error.",
   "depChecker.dotnetInstallErrorCode": "dotnet-install command failed.",
-  "depChecker.failToInstallBicepOutputVSC": "Unable to install @NameVersion. please read this wiki (@HelpLink) to install @NameVersion manually and restart Visual Studio Code.",
```

Comment:
I know the intention is to unify errors but if user manually installs bicep, restarting vscode after installation is mandatory (otherwise toolkit won't see the latest PATH environment variable after installation). How do we let user know it?

Reply:
Maybe append this information like this?
![image](https://user-images.githubusercontent.com/9698542/229413903-05b6220a-cb31-431b-83ef-14fe524da3fb.png)